Tests/FluentAssertions.Specs/Exceptions/ExceptionAssertionSpecs.cs

```diff
@@ -896,6 +896,20 @@ public void When_invoking_a_function_on_a_null_subject_it_should_throw()
                 .WithParameterName("subject");
         }
 
+        [Fact]
+        public void When_invoking_a_function_with_null_it_should_throw()
```

Comment:
Would be cool if you could start using a shorter naming convention, like `Invoking_with_null_is_not_allowed`